### PR TITLE
CreateImageWizard: single upload_request

### PR DESCRIPTION
**Pull request by chloenayon**
Change image_request to accept a single upload_request instead of
an array of upload_requests.

### CODE REVIEW

**Comment by Gundersanne** [comment]
Nice! Adding blocked until the related composer PR is merged (osbuild/osbuild-composer#1267)

File: src/SmartComponents/CreateImageWizard/CreateImageWizard.js [hunk 1]
```diff
@@ -259,7 +259,7 @@ class CreateImageWizard extends Component {
                     {
                         architecture: this.state.arch,
                         image_type: 'ami',
-                        upload_requests: [ this.state.uploadAWS ],
+                        upload_request: [ this.state.uploadAWS ],
```
Comment (Gundersanne) [inline src/SmartComponents/CreateImageWizard/CreateImageWizard.js:262]: This needs to not be an array now

File: src/SmartComponents/CreateImageWizard/CreateImageWizard.js [hunk 2]
```diff
@@ -278,7 +278,7 @@ class CreateImageWizard extends Component {
                     {
                         architecture: this.state.arch,
                         image_type: 'gcp',
-                        upload_requests: [ upload_google ],
+                        upload_request: [ upload_google ],
```
Comment (Gundersanne) [inline src/SmartComponents/CreateImageWizard/CreateImageWizard.js:281]: ditto

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Because of #103  this needs a rebase now

**Comment by chloenayon** [comment]
made fixes to address @Gundersanne 's comments

**Review by Gundersanne — APPROVED** [review]

**Pull request merged**